test/Crate/Test/DBAL/Platforms/CratePlatformTest.php

```diff
@@ -67,25 +67,36 @@ public function getGenerateUniqueIndexSql()
     public function testGeneratesForeignKeyCreationSql()
     {
         $fk = new \Doctrine\DBAL\Schema\ForeignKeyConstraint(array('fk_name_id'), 'other_table', array('id'), '');
-    
+
         $this->assertEquals(
             $this->getGenerateForeignKeySql(),
             $this->_platform->getCreateForeignKeySQL($fk, 'test')
         );
     }
-    
+
     public function getGenerateForeignKeySql()
     {
         $this->markTestSkipped('Platform does not support ADD FOREIGN KEY.');
     }
 
+    public function testGeneratesDecimalTypeDeclarationSQL()
+    {
+        $this->markTestSkipped('Platform does not support any decleration of datatype DECIMAL.');
```

Comment:
why is this related to the 2.6 upgrade?

Reply:
this test method needs to get an override otherwise it would test datatypes `DECIMAL` which Crate does not support